src/Client/ClientBase.cpp

```diff
@@ -1356,6 +1364,17 @@ void ClientBase::runInteractive()
             has_vertical_output_suffix = true;
         }
 
+        for (const auto& [alias, command] : backslash_aliases)
+        {
+            if (input.starts_with(alias))
```

Comment:
I've checked that `psql` supports these escape sequences even if you put extra whitespace before:

```
$ sudo -u postgres psql tutorial
psql (13.4 (Ubuntu 13.4-4.pgdg18.04+1))
Type "help" for help.

tutorial=#  \d
                List of relations
 Schema |         Name         | Type  |  Owner   
--------+----------------------+-------+----------
 public | hits_100m_obfuscated | table | postgres
(1 row)

tutorial=#        \d
                List of relations
 Schema |         Name         | Type  |  Owner   
--------+----------------------+-------+----------
 public | hits_100m_obfuscated | table | postgres
(1 row)
```

Reply:
Also let's require whitespace separator to avoid accepting something like

`\da`

PS. I just noticed, that `\da` in PostgreSQL is listing aggregate functions :)

Reply:
Enhanced in 7b50ab7